plugin.video.mlbbasesloaded/resources/language/English/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
can you use the new strings.po system (http://kodi.wiki/view/language_support). Please also consider to use the new folder layout (see here: https://github.com/xbmc/generator-kodi-addon/tree/master/generators/app/templates/resources/language). Also please use the correct id's (strings 30000 thru 30999 reserved for plugins and plugin settings)